src/components/lobby/LobbySeeks.tsx

```diff
@@ -1,15 +1,77 @@
+// tslint:disable:no-console
 import * as React from 'react';
-
-class LobbySeeksComponent extends React.Component<any, any> {
+import styled from 'styled-components';
+import {CreatedGames} from '../../store/MockData';
+const Container = styled.div `
+grid-column-start: 2;
+grid-column-end: 3;
+grid-row-start: 2;
+grid-row-end: 3;
+background-color: white;
+opacity: 0.8;
+height: auto;
+width: auto;
+overflow-y: scroll;
+padding: 50px 40px 0px 40px;
```

Comment:
There is an excessive amount of padding on the top here, might trim it down a bit, no more than 30px (instead of 50) on the top is my recommendation.